src/cluster/listCluster.tsx

```diff
@@ -214,23 +215,23 @@ function ListCluster({
       <div
         role="button"
         aria-disabled={
-          data.status.state !== ClusterStatus.STATUS_STOPPED &&
+          ClusterStatusState[data.status.state.toString()] !== ClusterStatus.STATUS_STOPPED &&
```

Comment:
Drop all of the `toString()` calls and instead use integer keys for the dictionary.